easy-to-use-api/klips-url-generator/src/App.tsx

```diff
@@ -108,50 +111,50 @@ const App: React.FC = () => {
           region={region}
           wktGeom={wktGeom}
           onDrawEnd={onDrawEnd}
-          // onDrawStart={onDrawStart}
         />
       case 'video':
         return <VideoComponent
           geoJsonGeom={geoJsonGeom}
           region={region}
           onDrawEnd={onDrawEnd}
-          // onDrawStart={onDrawStart}
         />
       case 'warning':
         return <WarningComponent
           wktGeom={wktGeom}
           geoJsonGeom={geoJsonGeom}
           region={region}
           onDrawEnd={onDrawEnd}
-          // onDrawStart={onDrawStart}
         />
     }
   }
 
   return (
     <div className='App'>
       <MapContext.Provider value={map}>
-        <div className='output-wrapper'>
-          <div className='header'>
-            <img src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
-            <h2>Widget URL Generator</h2>
+        <ConfigProvider 
+        >
+          <div className='output-wrapper'>
+            <div className='header'>
+              <img className='logo' src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
```

Comment:
just reminder, we should host this on our server

Reply:
Good point!